### ISSUE TITLE
Always getting password authentication fail

### ISSUE THREAD

**Issue by Lotiara** [timescale/timescaledb-parallel-copy#18]
Hi,
I am always getting a password authentication fail, no matter
which account (postgres or others).
I know you can specify the --connection option, but no password can be passed as a parameter.
Wondering what I am doing wrong.
Bye and good job.

**Comment by jd0x** [comment]
I am receiving this as well, however, my issue appears to be tied to the lack of custom port support. I provided my command and response below. 

```bash
CONNECTION="host=$DB_HOST user=$DB_USER password=$DB_PASSWORD port=$DB_PORT sslmode=require"
COLUMNS="active"
timescaledb-parallel-copy --db-name main \
  --table mytable \
  --connection $CONNECTION --reporting-period 2s --columns $COLUMNS \
  --copy-options "CSV NULL ''" --workers 1 --file /tmp/data.csv
```

Error log provided below with IP address redacted.

```bash
panic: dial tcp x.x.x.x:5432: connect: connection timed out
goroutine 19 [running]:
github.com/jmoiron/sqlx.MustConnect(...)
        /go/src/github.com/jmoiron/sqlx/sqlx.go:650
main.processBatches(0xc00008a2a4, 0xc00008e1e0)
        /go/src/github.com/timescale/timescaledb-parallel-copy/cmd/timescaledb-parallel-copy/main.go:206 +0xd2f
created by main.main
        /go/src/github.com/timescale/timescaledb-parallel-copy/cmd/timescaledb-parallel-copy/main.go:124 +0x181
```

**Comment by bboule** [comment]
@blagojts could you have a look?

**Comment by blagojts** [comment]
I have confirmed your issues, there seems to be a bug, will update the connection options and let you know. Thank you for reporting

**Comment by jd0x** [comment]
Sorry for the delay @blagojts, I was traveling last week. 

I received the same error as I shared above executing your script. Your testing environment is using the default postgres port (5432). The issue I am running into is due to a custom, non-default postgres port being used. 

Do you mind trying this on a custom port and reporting back?

**Comment by RobAtticus** [comment]
Hi @Lotiara and others:

I have made a PR here https://github.com/timescale/timescaledb-parallel-copy/pull/20 that will hopefully make this kind of thing easier to do. You can set appropriate environment variables for password, port, whatever like you would with `psql` and it should end up working.

**Comment by RobAtticus** [comment]
@jd0x and @Lotiara this has been merged with #20.
